### ISSUE TITLE
Exception raised when a wrong auth token provided

### ISSUE THREAD

**Issue by laoyur** [JamesRitchie/django-rest-framework-expiring-tokens#11]
[https://github.com/JamesRitchie/django-rest-framework-expiring-tokens/blob/master/rest_framework_expiring_authtoken/authentication.py#L27](url)

authentication.py line 27:
`except self.model.DoesNotExist:` should be `except self.models.DoesNotExist:` note the missing `s` of `models`

**Comment by max-pfeiffer** [comment]
Same problem here:

`File "/Users/max/python_environments/server/lib/python3.4/site-packages/rest_framework_expiring_authtoken/authentication.py", line 27, in authenticate_credentials
    except self.model.DoesNotExist:
AttributeError: 'NoneType' object has no attribute 'DoesNotExist'
`

**Comment by JamesRitchie** [comment]
Fixed with #12

**Comment by hwkns** [comment]
Thanks for fixing this!  For me, the error was getting masked by:
```python
AttributeError: 'WSGIRequest' object has no attribute 'successful_authenticator'
```
Took a while to find the root cause.  Just leaving this here in case others are searching for that same error string.
